### PR TITLE
Migrate pages and set up redirects

### PR DESCRIPTION
**Pull request by cbrownstein-lambda**
## Description

This PR:

- Sets up redirects for the migration.
- Migrates pages.

## Related issues

None

## Checklist

- [ ] ~~Documentation follows the [contributing guidelines](../CONTRIBUTING.md).~~
- [ ] ~~Documentation linked to in nav and index pages.~~
- [ ] ~~Files used in this PR (e.g., images and PDFs) are committed.~~
- [ ] ~~Link to existing documentation and the reverse.~~
- [x] Changes have been tested locally.
- [x] Reviewers have been assigned (at least 1 tech writer and 1 eng).

## Additional notes

The redirects are believed to be but might not be 100% up to date.

@cbrownstein-lambda

- [x] tags.md
- [x] index.md
- [x] education/programming/virtual-environments-containers.md
- [x] education/programming/vs-code-lambda-chat.md
- [x] education/large-language-models/serving-llama-3-1-docker.md
- [x] education/large-language-models/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md
- [ ] education/large-language-models/deploying-a-llama-3-inference-endpoint.md
- [x] education/large-language-models/lambda-chat-api.md
- [x] education/large-language-models/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md
- [x] education/large-language-models/serving-llama-3-1-405b.md
- [x] education/index.md
- [x] education/tutorial-getting-started-with-training-a-machine-learning-model.md
- [x] education/generative-ai/flux-prompt-to-image.md
- [x] education/linux-usage/using-the-lambda-bug-report-to-troubleshoot-your-system.md
- [x] education/linux-usage/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md
- [x] education/linux-usage/basic-linux-commands-and-system-administration.md
- [x] education/linux-usage/troubleshooting-and-debugging.md
- [x] education/linux-usage/lambda-stack-and-recovery-images.md
- [x] education/linux-usage/configuring-software-raid.md
- [x] education/scheduling-and-orchestration/deploying-models-with-dstack.md
- [x] education/scheduling-and-orchestration/skypilot-deploy-kubernetes.md
- [x] private-cloud/managed-kubernetes/index.md
- [x] private-cloud/managed-kubernetes/getting-started.md
- [x] public-cloud/filesystems.md
- [x] public-cloud/on-demand/demos.md

@ta11t0m

- [x] public-cloud/on-demand/index.md
- [x] public-cloud/on-demand/billing.md
- [x] public-cloud/on-demand/getting-started.md
- [x] public-cloud/on-demand/dashboard.md
- [x] public-cloud/on-demand/guest-agent.md
- [x] public-cloud/teams.md
- [x] public-cloud/firewalls.md
- [x] public-cloud/cloud-api.md
- [x] public-cloud/index.md
- [x] public-cloud/1-click-clusters/serving-llama-3_1-405b.md
- [x] public-cloud/1-click-clusters/getting-started.md
- [x] public-cloud/1-click-clusters/security-posture.md
- [x] hardware/tensorbook/getting-started.md
- [x] hardware/workstations/troubleshooting.md
- [x] hardware/workstations/getting-started.md
- [x] hardware/servers/unboxing-and-setup/using-bmc-ipmi.md
- [x] hardware/servers/unboxing-and-setup/power-and-network-cables.md
- [x] hardware/servers/getting-started.md
- [ ] hardware/index.md

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/using-skypilot-to-deploy-a-kubernetes-cluster.md': 'education/scheduling-and-orchestration/skypilot-deploy-kubernetes.md'
+        'on-demand-cloud/team.md': 'public-cloud/team.md'
+        'on-demand-cloud/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md': 'education/large-language-models/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md'
+        'on-demand-cloud/integrating-lambda-chat-into-vs-code.md': 'education/programming/vs-code-lambda-chat.md'
+        'on-demand-cloud/how-to-serve-the-flux.1-prompt-to-image-models-using-lambda-cloud-on-demand-instances.md': 'education/generative-ai/flux-prompt-to-image.md'
+        'on-demand-cloud/firewall.md': 'public-cloud/firewall.md'
+        'on-demand-cloud/cloud-api.md': 'public-cloud/cloud-api.md'
+        'on-demand-cloud/overview.md': 'public-cloud/on-demand/overview.md'
+        'on-demand-cloud/demos.md': 'public-cloud/on-demand/demos.md'
+        'on-demand-cloud/tutorial-getting-started-with-training-a-machine-learning-model.md': 'education/tutorial-getting-started-with-training-a-machine-learning-model.md'
```
Comment (ryanlambda) [inline mkdocs.yml:126]: has this been migrated?

File: mkdocs.yml [hunk 2]
```diff
@@ -114,6 +114,38 @@ plugins:
...
         'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/using-skypilot-to-deploy-a-kubernetes-cluster.md': 'education/scheduling-and-orchestration/skypilot-deploy-kubernetes.md'
+        'on-demand-cloud/team.md': 'public-cloud/team.md'
+        'on-demand-cloud/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md': 'education/large-language-models/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md'
+        'on-demand-cloud/integrating-lambda-chat-into-vs-code.md': 'education/programming/vs-code-lambda-chat.md'
+        'on-demand-cloud/how-to-serve-the-flux.1-prompt-to-image-models-using-lambda-cloud-on-demand-instances.md': 'education/generative-ai/flux-prompt-to-image.md'
+        'on-demand-cloud/firewall.md': 'public-cloud/firewall.md'
+        'on-demand-cloud/cloud-api.md': 'public-cloud/cloud-api.md'
+        'on-demand-cloud/overview.md': 'public-cloud/on-demand/overview.md'
+        'on-demand-cloud/demos.md': 'public-cloud/on-demand/demos.md'
```
Comment (ryanlambda) [inline mkdocs.yml:125]: has this been migrated?

File: mkdocs.yml [hunk 3]
```diff
@@ -114,6 +114,38 @@ plugins:
       redirect_maps:
         'one-click-clusters/getting-started.md': 'public-cloud/1-click-clusters/getting-started.md'
         'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/using-skypilot-to-deploy-a-kubernetes-cluster.md': 'education/scheduling-and-orchestration/skypilot-deploy-kubernetes.md'
+        'on-demand-cloud/team.md': 'public-cloud/team.md'
+        'on-demand-cloud/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md': 'education/large-language-models/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md'
```
Comment (ryanlambda) [inline mkdocs.yml:119]: not yet migrated

File: mkdocs.yml [hunk 4]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/cloud-api.md': 'public-cloud/cloud-api.md'
+        'on-demand-cloud/overview.md': 'public-cloud/on-demand/overview.md'
+        'on-demand-cloud/demos.md': 'public-cloud/on-demand/demos.md'
+        'on-demand-cloud/tutorial-getting-started-with-training-a-machine-learning-model.md': 'education/tutorial-getting-started-with-training-a-machine-learning-model.md'
+        'on-demand-cloud/deploying-a-llama-3-inference-endpoint.md': 'education/large-language-models/deploying-a-llama-3-inference-endpoint.md'
+        'on-demand-cloud/using-the-lambda-chat-completions-api.md': 'education/large-language-models/lambda-chat-api.md'
+        'on-demand-cloud/billing.md': 'public-cloud/billing.md'
+        'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/dashboard.md': 'public-cloud/dashboard.md'
+        'on-demand-cloud/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md': 'education/large-language-models/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md'
```
Comment (ryanlambda) [inline mkdocs.yml:132]:
```suggestion
        'on-demand-cloud/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md': 'education/large-language-models/serving-llama-3-1-docker.md'
```

File: mkdocs.yml [hunk 5]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/deploying-models-with-dstack.md': 'education/scheduling-and-orchestration/deploying-models-with-dstack.md'
+        'on-demand-cloud/file-systems.md': 'public-cloud/file-systems.md'
+        'tensorbook/getting-started.md': 'hardware/tensorbook/getting-started.md'
+        'workstations/troubleshooting-workstations-and-desktops.md': 'hardware/workstations/troubleshooting.md'
+        'workstations/getting-started.md': 'hardware/workstations/getting-started.md'
+        'software/virtual-environments-and-docker-containers.md': 'education/programming/virtual-environments-containers.md'
+        'software/using-the-lambda-bug-report-to-troubleshoot-your-system.md': 'education/linux-usage/using-the-lambda-bug-report-to-troubleshoot-your-system.md'
```
Comment (ryanlambda) [inline mkdocs.yml:139]: not migrated afaict

File: mkdocs.yml [hunk 6]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'software/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md': 'education/linux-usage/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md'
```
Comment (ryanlambda) [inline mkdocs.yml:140]: not migrated afaict

File: mkdocs.yml [hunk 7]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'software/basic-linux-commands-and-system-administration.md': 'education/linux-usage/basic-linux-commands-and-system-administration.md'
+        'software/troubleshooting-and-debugging.md': 'education/linux-usage/troubleshooting-and-debugging.md'
```
Comment (ryanlambda) [inline mkdocs.yml:142]: not migrated yet

File: mkdocs.yml [hunk 8]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'software/lambda-stack-and-recovery-images.md': 'education/linux-usage/lambda-stack-and-recovery-images.md'
```
Comment (ryanlambda) [inline mkdocs.yml:143]: not migrated yet

File: mkdocs.yml [hunk 9]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'software/configuring-software-raid.md': 'education/linux-usage/configuring-software-raid.md'
```
Comment (ryanlambda) [inline mkdocs.yml:144]: not yet migrated

File: mkdocs.yml [hunk 10]
```diff
@@ -114,6 +114,38 @@ plugins:
       redirect_maps:
         'one-click-clusters/getting-started.md': 'public-cloud/1-click-clusters/getting-started.md'
         'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/using-skypilot-to-deploy-a-kubernetes-cluster.md': 'education/scheduling-and-orchestration/skypilot-deploy-kubernetes.md'
+        'on-demand-cloud/team.md': 'public-cloud/team.md'
```
Comment (ryanlambda) [inline mkdocs.yml:118]:
```suggestion
        'on-demand-cloud/team.md': 'public-cloud/teams.md'
```

File: mkdocs.yml [hunk 11]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md': 'education/large-language-models/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md'
+        'on-demand-cloud/integrating-lambda-chat-into-vs-code.md': 'education/programming/vs-code-lambda-chat.md'
+        'on-demand-cloud/how-to-serve-the-flux.1-prompt-to-image-models-using-lambda-cloud-on-demand-instances.md': 'education/generative-ai/flux-prompt-to-image.md'
+        'on-demand-cloud/firewall.md': 'public-cloud/firewall.md'
+        'on-demand-cloud/cloud-api.md': 'public-cloud/cloud-api.md'
+        'on-demand-cloud/overview.md': 'public-cloud/on-demand/overview.md'
```
Comment (ryanlambda) [inline mkdocs.yml:124]:
```suggestion
        'on-demand-cloud/overview.md': 'public-cloud/on-demand/index.md'
```

File: mkdocs.yml [hunk 12]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/tutorial-getting-started-with-training-a-machine-learning-model.md': 'education/tutorial-getting-started-with-training-a-machine-learning-model.md'
+        'on-demand-cloud/deploying-a-llama-3-inference-endpoint.md': 'education/large-language-models/deploying-a-llama-3-inference-endpoint.md'
+        'on-demand-cloud/using-the-lambda-chat-completions-api.md': 'education/large-language-models/lambda-chat-api.md'
+        'on-demand-cloud/billing.md': 'public-cloud/billing.md'
+        'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/dashboard.md': 'public-cloud/dashboard.md'
+        'on-demand-cloud/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md': 'education/large-language-models/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md'
+        'on-demand-cloud/deploying-models-with-dstack.md': 'education/scheduling-and-orchestration/deploying-models-with-dstack.md'
+        'on-demand-cloud/file-systems.md': 'public-cloud/file-systems.md'
+        'tensorbook/getting-started.md': 'hardware/tensorbook/getting-started.md'
```
Comment (ryanlambda) [inline mkdocs.yml:135]: not migrated yet

File: mkdocs.yml [hunk 13]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/demos.md': 'public-cloud/on-demand/demos.md'
+        'on-demand-cloud/tutorial-getting-started-with-training-a-machine-learning-model.md': 'education/tutorial-getting-started-with-training-a-machine-learning-model.md'
+        'on-demand-cloud/deploying-a-llama-3-inference-endpoint.md': 'education/large-language-models/deploying-a-llama-3-inference-endpoint.md'
+        'on-demand-cloud/using-the-lambda-chat-completions-api.md': 'education/large-language-models/lambda-chat-api.md'
+        'on-demand-cloud/billing.md': 'public-cloud/billing.md'
+        'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/dashboard.md': 'public-cloud/dashboard.md'
+        'on-demand-cloud/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md': 'education/large-language-models/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md'
+        'on-demand-cloud/deploying-models-with-dstack.md': 'education/scheduling-and-orchestration/deploying-models-with-dstack.md'
+        'on-demand-cloud/file-systems.md': 'public-cloud/file-systems.md'
```
Comment (ryanlambda) [inline mkdocs.yml:134]:
```suggestion
        'on-demand-cloud/file-systems.md': 'public-cloud/filesystems.md'
```

File: mkdocs.yml [hunk 14]
```diff
@@ -114,6 +114,38 @@ plugins:
       redirect_maps:
         'one-click-clusters/getting-started.md': 'public-cloud/1-click-clusters/getting-started.md'
         'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/using-skypilot-to-deploy-a-kubernetes-cluster.md': 'education/scheduling-and-orchestration/skypilot-deploy-kubernetes.md'
+        'on-demand-cloud/team.md': 'public-cloud/team.md'
+        'on-demand-cloud/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md': 'education/large-language-models/deploying-llama-3.2-3b-in-a-kubernetes-k8s-cluster.md'
+        'on-demand-cloud/integrating-lambda-chat-into-vs-code.md': 'education/programming/vs-code-lambda-chat.md'
+        'on-demand-cloud/how-to-serve-the-flux.1-prompt-to-image-models-using-lambda-cloud-on-demand-instances.md': 'education/generative-ai/flux-prompt-to-image.md'
+        'on-demand-cloud/firewall.md': 'public-cloud/firewall.md'
```
Comment (ryanlambda) [inline mkdocs.yml:122]:
```suggestion
        'on-demand-cloud/firewall.md': 'public-cloud/firewalls.md'
```

File: mkdocs.yml [hunk 15]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/cloud-api.md': 'public-cloud/cloud-api.md'
+        'on-demand-cloud/overview.md': 'public-cloud/on-demand/overview.md'
+        'on-demand-cloud/demos.md': 'public-cloud/on-demand/demos.md'
+        'on-demand-cloud/tutorial-getting-started-with-training-a-machine-learning-model.md': 'education/tutorial-getting-started-with-training-a-machine-learning-model.md'
+        'on-demand-cloud/deploying-a-llama-3-inference-endpoint.md': 'education/large-language-models/deploying-a-llama-3-inference-endpoint.md'
```
Comment (ryanlambda) [inline mkdocs.yml:127]: not yet migrated

File: mkdocs.yml [hunk 16]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/using-the-lambda-chat-completions-api.md': 'education/large-language-models/lambda-chat-api.md'
+        'on-demand-cloud/billing.md': 'public-cloud/billing.md'
+        'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/dashboard.md': 'public-cloud/dashboard.md'
+        'on-demand-cloud/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md': 'education/large-language-models/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md'
+        'on-demand-cloud/deploying-models-with-dstack.md': 'education/scheduling-and-orchestration/deploying-models-with-dstack.md'
```
Comment (ryanlambda) [inline mkdocs.yml:133]:
```suggestion
        'on-demand-cloud/deploying-models-with-dstack.md': 'education/scheduling-and-orchestration/dstack-vllm-llama-3-1.md'
```

File: mkdocs.yml [hunk 17]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/file-systems.md': 'public-cloud/file-systems.md'
+        'tensorbook/getting-started.md': 'hardware/tensorbook/getting-started.md'
+        'workstations/troubleshooting-workstations-and-desktops.md': 'hardware/workstations/troubleshooting.md'
+        'workstations/getting-started.md': 'hardware/workstations/getting-started.md'
+        'software/virtual-environments-and-docker-containers.md': 'education/programming/virtual-environments-containers.md'
+        'software/using-the-lambda-bug-report-to-troubleshoot-your-system.md': 'education/linux-usage/using-the-lambda-bug-report-to-troubleshoot-your-system.md'
+        'software/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md': 'education/linux-usage/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md'
+        'software/basic-linux-commands-and-system-administration.md': 'education/linux-usage/basic-linux-commands-and-system-administration.md'
```
Comment (ryanlambda) [inline mkdocs.yml:141]: not yet migrated

File: mkdocs.yml [hunk 18]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'software/troubleshooting-and-debugging.md': 'education/linux-usage/troubleshooting-and-debugging.md'
+        'software/lambda-stack-and-recovery-images.md': 'education/linux-usage/lambda-stack-and-recovery-images.md'
+        'software/configuring-software-raid.md': 'education/linux-usage/configuring-software-raid.md'
+        'servers/getting-started.md': 'hardware/servers/getting-started.md'
+        '1-click-clusters/serving-llama-3.1-405b-on-a-lambda-1-click-cluster.md': 'education/large-language-models/serving-llama-3-1-405b.md'
```
Comment (ryanlambda) [inline mkdocs.yml:146]: currently this file is called "serving-llama-3_1-405b.md", but i support changing the underscore to a hyphen

File: mkdocs.yml [hunk 19]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'workstations/getting-started.md': 'hardware/workstations/getting-started.md'
+        'software/virtual-environments-and-docker-containers.md': 'education/programming/virtual-environments-containers.md'
+        'software/using-the-lambda-bug-report-to-troubleshoot-your-system.md': 'education/linux-usage/using-the-lambda-bug-report-to-troubleshoot-your-system.md'
+        'software/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md': 'education/linux-usage/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md'
+        'software/basic-linux-commands-and-system-administration.md': 'education/linux-usage/basic-linux-commands-and-system-administration.md'
+        'software/troubleshooting-and-debugging.md': 'education/linux-usage/troubleshooting-and-debugging.md'
+        'software/lambda-stack-and-recovery-images.md': 'education/linux-usage/lambda-stack-and-recovery-images.md'
+        'software/configuring-software-raid.md': 'education/linux-usage/configuring-software-raid.md'
+        'servers/getting-started.md': 'hardware/servers/getting-started.md'
```
Comment (ryanlambda) [inline mkdocs.yml:145]: not yet migrated

File: mkdocs.yml [hunk 20]
```diff
@@ -114,6 +114,38 @@ plugins:
...
+        'on-demand-cloud/billing.md': 'public-cloud/billing.md'
+        'on-demand-cloud/getting-started.md': 'public-cloud/on-demand/getting-started.md'
+        'on-demand-cloud/dashboard.md': 'public-cloud/dashboard.md'
+        'on-demand-cloud/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md': 'education/large-language-models/how-to-serve-the-llama-3.1-8b-and-70b-models-using-lambda-cloud-on-demand-instances.md'
+        'on-demand-cloud/deploying-models-with-dstack.md': 'education/scheduling-and-orchestration/deploying-models-with-dstack.md'
+        'on-demand-cloud/file-systems.md': 'public-cloud/file-systems.md'
+        'tensorbook/getting-started.md': 'hardware/tensorbook/getting-started.md'
+        'workstations/troubleshooting-workstations-and-desktops.md': 'hardware/workstations/troubleshooting.md'
+        'workstations/getting-started.md': 'hardware/workstations/getting-started.md'
+        'software/virtual-environments-and-docker-containers.md': 'education/programming/virtual-environments-containers.md'
```
Comment (ryanlambda) [inline mkdocs.yml:138]:
```suggestion
        'software/virtual-environments-and-docker-containers.md': 'education/linux-usage/virtual-environments-containers.md'
```

**Review by ryanlambda — CHANGES_REQUESTED** [review]
We'll need to migrate most of these before I can verify the redirects.
Also: added some suggestions to resolve a few issues with path and name changes.

File: mkdocs.yml (hunk 10 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:118]: "Team" is (was?) considered a feature rather than a common noun. Thoughts?

File: mkdocs.yml (hunk 2 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:125]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 3 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:119]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 5 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:139]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 6 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:140]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 7 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:142]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 8 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:143]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 9 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:144]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 12 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:135]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 15 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:127]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 17 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:141]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml (hunk 19 above)
Comment (cbrownstein-lambda) [reply mkdocs.yml:145]: Pages migrated in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/35/commits/f9df135e440a0fb5237cdc624f3418abb2f5301b.

File: mkdocs.yml [hunk 21]
```diff
@@ -114,6 +114,38 @@ plugins:
       redirect_maps:
         'one-click-clusters/getting-started.md': 'public-cloud/1-click-clusters/getting-started.md'
```
Comment (ta11t0m) [inline mkdocs.yml:115]: the destination path will need to change to /index.md once #40 is submitted.

File: docs/education/large-language-models/deploying-a-llama-3-inference-endpoint.md [hunk 22]
```diff
@@ -0,0 +1,166 @@
...
+| 8B (8 billion parameters) | More efficient and accessible, suitable for tasks where resources are constrained. The 8B model requires a 1x A100 or H100 GPU node. |
+| 70B (70 billion parameters) | Superior performance and capabilities ideal for complex or high-stakes applications. The 70B model requires an 8x A100 or H100 GPU node. |
+
+### Prerequisites
+
+This tutorial assumes the following prerequisites:
+
+1. Lambda On-Demand Cloud instances appropriate for the Llama 3 model size you want to run.
+   * Model 8B ([meta-llama/Meta-Llama-3-8B)](https://huggingface.co/meta-llama/Meta-Llama-3-8B) requires 1x A100 or H100 GPU node.
+   * Model 70B ([meta-llama/Meta-Llama-3-70B)](https://huggingface.co/meta-llama/Meta-Llama-3-70B) requires 8x A100 or H100 GPU nodes.
```
Comment (cbrownstein-lambda) [inline docs/education/large-language-models/deploying-a-llama-3-inference-endpoint.md:26]:
```suggestion
    * Model 8B ([meta-llama/Meta-Llama-3-8B)](https://huggingface.co/meta-llama/Meta-Llama-3-8B) requires 1x A100 or H100 GPU node.
    * Model 70B ([meta-llama/Meta-Llama-3-70B)](https://huggingface.co/meta-llama/Meta-Llama-3-70B) requires 8x A100 or H100 GPU nodes.
```

File: docs/education/large-language-models/deploying-a-llama-3-inference-endpoint.md [hunk 23]
```diff
@@ -0,0 +1,166 @@
...
+    === "Llama3 8B"
+
+        ```
+        python3 -m vllm.entrypoints.openai.api_server \
+          --host=0.0.0.0 \
+          --port=8000 \
+          --model=meta-llama/Meta-Llama-3-8B &> api_server.log & 
+        ```
+
+    ==="Llama3 70B"
```
Comment (cbrownstein-lambda) [inline docs/education/large-language-models/deploying-a-llama-3-inference-endpoint.md:74]:
```suggestion
    === "Llama3 70B"
```

File: docs/public-cloud/firewalls.md [hunk 24]
```diff
@@ -72,17 +81,17 @@ To update a firewall rule:
 
 To delete a firewall rule:
```
Comment (ryanlambda) [inline docs/public-cloud/firewalls.md:83]: Nit: I'd argue this is better placed before the steps. If you place it after the steps, the user might already have made the mistake it's warning about.

File: docs/education/programming/virtual-environments-containers.md [hunk 25]
```diff
@@ -0,0 +1,290 @@
+---
+description: Learn how to create a Python or conda virtual development environment
+tags:
+  - Virtualization
```
Comment (ryanlambda) [inline docs/education/programming/virtual-environments-containers.md:4]:
```suggestion
  - virtualization
```

Rationale: Consistency. All of the other tags are lower-case.

**Review by ryanlambda — APPROVED** [review]
A few minor nits. I saw some other more major issues, but the current state of things makes it hard to tell if they're already going to be solved in subsequent PRs. I'm just going to assume they are and approve for now, and we can do a QA check after the dust has settled.

**Review by ta11t0m — APPROVED** [review]

**Pull request merged**